sebsync.py

```diff
@@ -1,7 +1,10 @@
 """Synchronize Standard Ebooks catalog with local EPUB collection."""
 
 import click
+import os
+import pickle
```

Comment:
While this is a Python-based project, I'd prefer a quasi-human-readable format to store such data over pickling Python objects. It seems JSON would be a natural fit.